extension/src/background/messageListener/popupMessageListener.ts

```diff
@@ -555,12 +560,39 @@ export const popupMessageListener = (request: Request, sessionStore: Store) => {
   const importAccount = async () => {
     const { password, privateKey } = request;
     let sourceKeys;
+
+    let mnemonicPhrase = await getEncryptedTemporaryData({
```

Comment:
these changes are similar to the changes I added here for `add account`: https://github.com/stellar/freighter/pull/1827/commits/cb9d65c3308f6bc4968ef89c85d9a8764fb12f75

We're making sure that if our session has timed out, we log back in with the user's password before continuing